### DISCUSSION TITLE
How can I fix the error ‘java calibration failed, exit 4294967294’?

### DISCUSSION THREAD

**Discussion by Alejandro-Castellanos-Alonso** [activityMonitoring/biobankAccelerometerAnalysis#264] [Q&A] [answered]
Good afternoon,

I get this error when trying to run the instruction “accProcess data/sample-small.csv.gz”:
Error: java calibration failed, exit 4294967294

I have run the command from the “Troubleshooting” section: conda install -n accelerometer openjdk=8, but the same error keeps occurring.

I am using Anaconda Prompt (with administrator privileges) on the “Windows 11” operating system. How can I fix that error to be able to use the command “accProcess data/sample-small.csv.gz”?

Thank you very much.

Best regards,
Alejandro

**Comment by chanshing** [comment]
@Alejandro-Castellanos-Alonso could you share the file for me to try?

**Comment by chanshing** [comment, marked as answer]
@Alejandro-Castellanos-Alonso I suspect I know your issue. You're processing a custom CSV file and the headers and time format might be different. Can you try:
```console
accProcess data/sample1.csv --csvTimeFormat 'yyy-MM-dd HH:mm:ss.SSS' --csvTimeXYZTempColsIndex 0,1,2,3
```

I will know for you if you would share the first 3-5 lines of your CSV file.

**Reply by Alejandro-Castellanos-Alonso** [reply]
> @Alejandro-Castellanos-Alonso I suspect I know your issue. You're processing a custom CSV file and the headers and time format might be different. Can you try:
> 
> ```
> accProcess data/sample1.csv --csvTimeFormat 'yyy-MM-dd HH:mm:ss.SSS' --csvTimeXYZTempColsIndex 0,1,2,3
> ```
> 
> I will know for you if you would share the first 3-5 lines of your CSV file.

Good afternoon, I have tried to use the command ‘accProcess data/sample1.csv --csvTimeFormat “yyy-MM-dd HH:mm:ss.SSS” --csvTimeXYZTempColsIndex 0,1,2,3’ and I get the error: ‘accProcess: error: error: unrecognized arguments: HH:mm:ss.SSS’’.

The file I am using has been downloaded directly from the Tutorial available on the web (link in Usage). The download address is: http://gas.ndph.ox.ac.uk/aidend/accModels/sample-small.csv.gz

How can I solve the error?

Thank you very much.

Best regards,
Alejandro

**Comment by Alejandro-Castellanos-Alonso** [comment]
Good afternoon, I have tried to use the command ‘accProcess data/sample1.csv --csvTimeFormat “yyy-MM-dd HH:mm:ss.SSS” --csvTimeXYZTempColsIndex 0,1,2,3’ and I get the error: ‘accProcess: error: error: unrecognized arguments: HH:mm:ss.SSS’’.

The file I am using has been downloaded directly from the Tutorial available on the web (link in Usage). The download address is: http://gas.ndph.ox.ac.uk/aidend/accModels/sample-small.csv.gz

How can I solve the error?

Thank you very much.

Best regards,
Alejandro

**Reply by chanshing** [reply]
Are you using Windows? Can you try with single quotes?

```
accProcess data/sample1.csv --csvTimeFormat 'yyy-MM-dd HH:mm:ss.SSS' --csvTimeXYZTempColsIndex '0,1,2,3'
```

**Reply by Alejandro-Castellanos-Alonso** [reply]
> Are you using Windows? Can you try with single quotes?
> 
> ```
> accProcess data/sample1.csv --csvTimeFormat 'yyy-MM-dd HH:mm:ss.SSS' --csvTimeXYZTempColsIndex '0,1,2,3'
> ```

Good morning, yes, I am using Windows 11. I have tried the new command using the single quotes and I get the same error -> accProcess: error: unrecognized arguments: HH:mm:ss.SSS'.

How can I solve this problem?

Thank you very much.

Best regards,
Alejandro

**Reply by chanshing** [reply]
Have you tried double quotes?

**Reply by Alejandro-Castellanos-Alonso** [reply]
> Have you tried double quotes?

Good morning, yes, I have tried double quotes previously. But I got the same error.

How can I fix it?

Thank you very much.

Best regards,
Alejandro

**Reply by chanshing** [reply]
For some reason it is taking "yyy-MM-dd HH:mm:ss.SSS" as two different separate strings instead of one in your terminal, not sure why.
